package.json

```diff
@@ -23,7 +23,17 @@
         "title": "Restart Angular Language server",
         "category": "Angular"
       }
-    ]
+    ],
+    "configuration": {
+      "title": "Angular Language Service",
+      "properties": {
+        "angular.ngdk": {
```

Comment:
What is `ngdk`? Also, can you add to the description that this points to the Language Service in node modules or falls back on the bundled version by default?

Reply:
It's not a very intuitive name, I must agree, but I also think it's good to align with TypeScript.
TypeScript has a config named `tsdk`, which I assume is short for "TypeScript development kit"
https://github.com/microsoft/vscode/blob/f3d4a67c2dddf45cb1636f959ae47ce47ead60d2/extensions/typescript-language-features/package.json#L119-L127

The description here will generate a tooltip that explains the field. I don't think we should mention fallback to the bundled version, because the very point of specifying `ngdk` is to override that behavior.

Reply:
Is there anyway to mark it as optional, then? I just worry that some users may think this is a required configuration.

Reply:
With the default value set, it's already optional. Same with `typescript.tsdk`, by default it's null.